### PR TITLE
Lal/sdcosmo 1729 acces list scenario

### PR DESCRIPTION
**Pull request by lalepee**

### CODE REVIEW

File: scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt [hunk 1]
```diff
@@ -548,21 +547,17 @@ internal class ScenarioServiceImpl(
       workspaceId: String,
       scenarioId: String
   ): Scenario {
-    checkInternalResultDataServiceConfiguration()
-    return findScenarioById(
-        organizationId, workspaceId, scenarioId, withLastRun = true, withState = true)
+    return findScenarioById(organizationId, workspaceId, scenarioId, withState = true)
```
Comment (jreynard-code) [inline scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt:550]: Why did you remove `checkInternalResultDataServiceConfiguration` method here?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM
Just  little question ;)

File: scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt (hunk 1 above)
Comment (lalepee) [reply scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt:551]: And that's why we need review. That's a mistake.

**Review by jreynard-code — APPROVED** [review]
LGTM
